apps/server/src/server.ts

```diff
@@ -15,8 +13,8 @@ process.on('unhandledRejection', (reason, p) => {
 
 initDb()
   .then(() => {
-    const engineProcess = rootContainer.get(EngineProcess);
-    return boostrapEngine(config.defaultEngine.path, engineProcess);
+    // todo: pass function that calls EngineProcessDirector for process init?
```

Comment:
We do not to send the process start now as it is moved to the individual process runner services. We can update the definition of the `bootstrapEngine` method to ignore the second parameter